lib/code_corps/map_utils.ex

```diff
@@ -8,7 +8,6 @@ defmodule CodeCorps.MapUtils do
   def keys_to_string(map), do: stringify_keys(map)
 
   # Goes through a list and stringifies keys of any map member
-  defp stringify_keys(nil), do: nil
   defp stringify_keys(%DateTime{} = val), do: val
```

Comment:
Thanks for the review, guys. @begedin I may revisit this thread for your idea on matching against structs--that's clever!

L19 actually isn't a duplicate of L22--`stringify_keys` and `stringify_key` are (somewhat non-obviously) different fns (plural v. singular).

I'll push up a commit dropping L11. LMK if there's further to be done; otherwise I'll rebase against develop, re-push, and move on.